src/engine/engine_zero.c

```diff
@@ -143,6 +140,9 @@ void ocf_engine_zero_line(struct ocf_request *req)
 
 	ENV_BUG_ON(req->core_line_count != 1);
 
+	/* No hash bucket locking - this function must be called under global
```

Comment:
To what this comment applies? It isn't above any fcn call and it's not in fcn header.

Reply:
It applies to ocf_engine_zero_line. Typically all other engines lock hash buckets around this line. This very engine is unique, I wanted to explain this here. I have edited this comment to make this clear - let me know what you think.

Reply:
Yup, now it's clearer!